### PR TITLE
Add `/download/dmg` support

### PR DESCRIPTION
**Pull request by AndyBitz**
Fixes https://github.com/zeit/hazel/issues/61

This adds `/download/dmg` as a hidden route that can be used to download the latest `dmg` file, if there is any.

We don't add it as replacement for `/download/mac` to not break any clients and won't show it on the overview page since it's not a platform itself, our recommended file for mac stays `.zip`.

### CODE REVIEW

**Comment by Aarbel** [comment]
@leo everything's ok ?

File: package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
   },
   "lint-staged": {
     "*.js": [
-      "yarn test",
-      "prettier --single-quote --no-semi --write",
+      "yarn test && :",
```
Comment (AndyBitz) [inline package.json]: This was added to make sure the provided arguments from `lint-staged` don't cause any trouble.

See [this comment](https://github.com/okonet/lint-staged/issues/174#issuecomment-437468837) from the `lint-staged` repo.

**Comment by Aarbel** [comment]
👍

**Comment by itsnwa** [comment]
@AndyBitz When I try to access the `download/dmg` route I get: `No download available for your platform` 🤷‍♂️

**Comment by Aarbel** [comment]
It doesn't seem to work, cf https://github.com/zeit/hazel/pull/53#issuecomment-581848569

**Comment by Aarbel** [comment]
@leo is that working for you ?

**Pull request merged**